python/src/nanoarrow/iterator.py

```diff
@@ -278,6 +398,50 @@ def _iter1(self, offset, length):
         return self._struct_tuple_iter(offset, length)
 
 
+def _get_tzinfo(tz_string, strategy=None):
+    # We can handle UTC without any imports
+    if tz_string.upper() == "UTC":
+        try:
+            # Added in Python 3.11
+            from datetime import UTC
```

Comment:
This is an alias of `datetime.timezone.utc` (lower case), which might be available in older versions as well. So could simplify to use that